setup.py

```diff
@@ -12,6 +12,7 @@
                    'hpy/universal/src/ctx_module.c',
                    'hpy/universal/src/ctx_meth.c',
                    'hpy/universal/src/ctx_misc.c',
+                   'hpy/universal/src/debugmode.c',
```

Comment:
In my idea, the "debug mode" be something different, i.e. a full wrapper around the `ctx` which does many extra checks, including potentially expensive ones, and will . So, maybe we should call this file differently?